blog/_src/posts/2016-05-18-gradual-typing-across-the-spectrum.md

```diff
@@ -0,0 +1,181 @@
+    Title: Gradual Typing Across the Spectrum
+    Date: 2016-05-18T07:58:56
+    Tags: gradual typing, PI meeting, by Asumu Takikawa
+
+> Instead of being Pythonistas, Rubyists, or Racketeers we have to
+> be scientists. -- Matthias Felleisen
+
+Yesterday we hosted a PI meeting for the Gradual Typing Across the Spectrum
```

Comment:
Link to: [the gtp site](http://prl.ccs.neu.edu/gtp/)  or the [nsf page](http://www.nsf.gov/awardsearch/showAward?AWD_ID=1518844)?